[2025-08-18T20:15:03.111+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_ingestion manual__2025-08-18T20:14:54.103226+00:00 [queued]>
[2025-08-18T20:15:03.126+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_ingestion manual__2025-08-18T20:14:54.103226+00:00 [queued]>
[2025-08-18T20:15:03.128+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-18T20:15:03.147+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): data_ingestion> on 2025-08-18 20:14:54.103226+00:00
[2025-08-18T20:15:03.155+0000] {standard_task_runner.py:60} INFO - Started process 465 to run task
[2025-08-18T20:15:03.160+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_pipeline', 'data_ingestion', 'manual__2025-08-18T20:14:54.103226+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn_pipeline_dag.py', '--cfg-path', '/tmp/tmpzbjx8udp']
[2025-08-18T20:15:03.166+0000] {standard_task_runner.py:88} INFO - Job 32: Subtask data_ingestion
[2025-08-18T20:15:03.249+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_churn_pipeline.data_ingestion manual__2025-08-18T20:14:54.103226+00:00 [running]> on host 6cfe0da8a177
[2025-08-18T20:15:03.351+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_pipeline' AIRFLOW_CTX_TASK_ID='data_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T20:14:54.103226+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T20:14:54.103226+00:00'
[2025-08-18T20:15:03.354+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T20:15:03.357+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/Customer_Churn_Prediction_Pipeline && python src/ingestion.py']
[2025-08-18T20:15:03.371+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T20:15:04.431+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,430 INFO --- Data Ingestion Pipeline Started ---
[2025-08-18T20:15:04.446+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,445 INFO Starting local data ingestion from data/raw/train/Telco-Dataset-train.csv
[2025-08-18T20:15:04.498+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,498 INFO Read 5634 rows from data/raw/train/Telco-Dataset-train.csv
[2025-08-18T20:15:04.604+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,603 INFO Local training data ingested and saved to data/raw/training_data/18-08-2025/Telco-Dataset-train.csv (size: 776827 bytes)
[2025-08-18T20:15:04.613+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,612 INFO Starting remote data ingestion from https://raw.githubusercontent.com/yugipersonalspace/ML-Assignment-PS3/main/Telco-Dataset-test.csv
[2025-08-18T20:15:04.717+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,716 INFO Remote test data downloaded and saved to data/raw/API_test_data/18-08-2025/Telco-Dataset-test.csv (size: 196084 bytes)
[2025-08-18T20:15:04.720+0000] {subprocess.py:93} INFO - 2025-08-18 20:15:04,718 INFO --- Data Ingestion Pipeline Completed Successfully ---
[2025-08-18T20:15:04.807+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-18T20:15:04.841+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_churn_pipeline, task_id=data_ingestion, execution_date=20250818T201454, start_date=20250818T201503, end_date=20250818T201504
[2025-08-18T20:15:04.865+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-18T20:15:04.897+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
